### PR TITLE
Pract.01: Add 04-06 | Martin Gatev

### PR DESCRIPTION
**Pull request by MGatev**
Martin Gatev
SI 1 0MI0600143

### CODE REVIEW

File: Seminars/Sem.01/Pract.01/Martin Gatev/Task04.cpp [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+using std::cin;
+using std::cout;
+
+const int MAX_SIZE_NAME = 31;
+
+struct Student
+{
+	unsigned int FN;
+	char name[MAX_SIZE_NAME];
```
Comment (khatoni) [inline Seminars/Sem.01/Pract.01/Martin Gatev/Task04.cpp:11]: Alignment??

File: Seminars/Sem.01/Pract.01/Martin Gatev/Task06.cpp [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+int isInsideTheCircle(const double radius, const Point& p1)
+{
+	double distanceFromCenterToPoint = distanceFromZero(p1);
+
+	if (distanceFromCenterToPoint < radius)
+	{
+		return 1;
+	}
+	if (distanceFromCenterToPoint - radius <= 0.00001)
```
Comment (khatoni) [inline Seminars/Sem.01/Pract.01/Martin Gatev/Task06.cpp:68]: please do not use just numbers, write global constant epsilon

File: Seminars/Sem.01/Pract.01/Martin Gatev/Task06.cpp [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+}
+
+int isInsideTheCircle(const double radius, const Point& p1)
+{
+	double distanceFromCenterToPoint = distanceFromZero(p1);
+
+	if (distanceFromCenterToPoint < radius)
+	{
+		return 1;
+	}
```
Comment (khatoni) [inline Seminars/Sem.01/Pract.01/Martin Gatev/Task06.cpp:67]: You can combine this if and the last statement and just use one line ternary operator

**Review by khatoni — CHANGES_REQUESTED** [review]

File: Seminars/Sem.01/Pract.01/Martin Gatev/Task04.cpp [hunk 4]
```diff
@@ -0,0 +1,22 @@
+#include <iostream>
+
+using std::cin;
+using std::cout;
+
+const int MAX_SIZE_NAME = 31;
```
Comment (Justsvetoslavov) [inline Seminars/Sem.01/Pract.01/Martin Gatev/Task04.cpp:6]: 30 , not 31

File: Seminars/Sem.01/Pract.01/Martin Gatev/Task05.cpp [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+using std::cin;
+using std::cout;
+
+struct ComplexNumber
+{
+	double realPart;
+	double imgPart;
+};
+
+ComplexNumber sumOfComplexNumbers(const ComplexNumber& number1,
```
Comment (Justsvetoslavov) [inline Seminars/Sem.01/Pract.01/Martin Gatev/Task05.cpp:12]: format?

File: Seminars/Sem.01/Pract.01/Martin Gatev/Task05.cpp [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+								  const ComplexNumber& number2)
+{
+	ComplexNumber sum{};
+
+	sum.realPart = number1.realPart + number2.realPart;
+	sum.imgPart = number1.imgPart + number2.imgPart;
+
+	return sum;
```
Comment (Justsvetoslavov) [inline Seminars/Sem.01/Pract.01/Martin Gatev/Task05.cpp:20]: 1 liner

**Review by Justsvetoslavov — CHANGES_REQUESTED** [review]

**Pull request closed without merging**